### ISSUE TITLE
 SBG_DRIVER - SBG_DRIVER - [Init] Unable to initialize the interface - SBG_ERROR

### ISSUE THREAD

**Issue by lonlonago** [SBG-Systems/sbg_ros_driver#68]
[ERROR] [1594611759.696153539]: SBG_DRIVER - SBG_DRIVER - [Init] Unable to initialize the interface - SBG_ERROR

i have changed the port name to /dev/ttyUSB0 , and boundrate 115200, and i can get data from minicom and cutecom , but still get this error.

**Comment by lonlonago** [comment]
@mzembsbg   yes , i found the solve way, chmod 777 /dev/ttyUSB0,   is problem of permisson.

**Comment by mzembsbg** [comment]
Glad to hear. I'll close the issue then.
